### ISSUE TITLE
yarn create t3-app crashes my M1 mac.

### ISSUE THREAD

**Issue by matt123miller** [t3-oss/create-t3-app#57]
**Describe the bug**
When running `yarn create t3-app` and saying yes to everything my M1 mac eventually crashes while trying to install prisma. This has happened twice in a row now. I include M1 because it might be ARM mac specific.

**To Reproduce**

M1 Macbook Air
Installed through nvm
node 18.4.0
yarn 3.2.1

1. Run this '...'

`yarn create t3-app`

Say yes to everything.

Wait

2. See error

The terminal hung while installing prisma, eventually crashing my whole laptop. The emergency mac Force Quit Applications modal appeared, spinning beachball I closed the terminal, then everything else while my laptop crawled along, then it restarted itself.

I've included the error details I received on restart.

I'll retry this again tomorrow and record a video on my phone.

```
panic(cpu 0 caller 0xfffffe002eac3a94): userspace watchdog timeout: no successful checkins from WindowServer in 149 seconds
service: logd, total successful checkins since load (1218 seconds ago): 119, last successful checkin: 0 seconds ago
service: WindowServer, total successful checkins since load (1188 seconds ago): 101, last successful checkin: 149 seconds ago
service: opendirectoryd, total successful checkins since load (1218 seconds ago): 118, last successful checkin: 0 seconds ago

Debugger message: panic
Memory ID: 0xff
OS release type: User
OS version: 21F79
Kernel version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:29 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T8101
Fileset Kernelcache UUID: 6761FE24D814CDDD6BDA1ADB023146F8
Kernel UUID: B747B891-04EC-3617-95EC-05E037610520
iBoot version: iBoot-7459.121.3
secure boot?: YES
Paniclog version: 13
KernelCache slide: 0x00000000267fc000
KernelCache base:  0xfffffe002d800000
Kernel slide:      0x0000000026fa4000
Kernel text base:  0xfffffe002dfa8000
Kernel text exec slide: 0x000000002708c000
Kernel text exec base:  0xfffffe002e090000
mach_absolute_time: 0x6f651c1f0
Epoch Time:        sec       usec
  Boot    : 0x62b9e73a 0x000c5373
  Sleep   : 0x00000000 0x00000000
  Wake    : 0x00000000 0x00000000
  Calendar: 0x62b9ec13 0x0006859c

Zone info:
  Zone map: 0xfffffe10000b8000 - 0xfffffe30000b8000
  . VM    : 0xfffffe10000b8000 - 0xfffffe14ccd84000
  . RO    : 0xfffffe14ccd84000 - 0xfffffe166671c000
  . GEN0  : 0xfffffe166671c000 - 0xfffffe1b333e8000
  . GEN1  : 0xfffffe1b333e8000 - 0xfffffe20000b4000
  . GEN2  : 0xfffffe20000b4000 - 0xfffffe24ccd80000
  . GEN3  : 0xfffffe24ccd80000 - 0xfffffe2999a4c000
  . DATA  : 0xfffffe2999a4c000 - 0xfffffe30000b8000
  Metadata: 0xfffffe74f7e6c000 - 0xfffffe74ffe6c000
  Bitmaps : 0xfffffe74ffe6c000 - 0xfffffe7502a70000

CORE 0 recently retired instr at 0xfffffe002e2259d4
CORE 1 recently retired instr at 0xfffffe002e226fb0
CORE 2 recently retired instr at 0xfffffe002e226fb0
CORE 3 recently retired instr at 0xfffffe002e226fb0
CORE 4 recently retired instr at 0xfffffe002e226fb4
CORE 5 recently retired instr at 0xfffffe002e226fb4
CORE 6 recently retired instr at 0xfffffe002e226fb4
CORE 7 recently retired instr at 0xfffffe002e226fb4
CORE 0 PVH locks held: None
CORE 1 PVH locks held: None
CORE 2 PVH locks held: None
CORE 3 PVH locks held: None
CORE 4 PVH locks held: None
CORE 5 PVH locks held: None
CORE 6 PVH locks held: None
CORE 7 PVH locks held: None
CORE 0 is the one that panicked. Check the full backtrace for details.
CORE 1: PC=0xfffffe002e1d43a4, LR=0xfffffe002e1d3ba8, FP=0xfffffe750c56bd10
CORE 2: PC=0xfffffe002e21c628, LR=0xfffffe002e21c3cc, FP=0xfffffe750dcabe20
CORE 3: PC=0x00000001046cc2e0, LR=0x00000001046ccce4, FP=0x000000016b73d930
CORE 4: PC=0xfffffe002e11e9e0, LR=0xfffffe002e11e9dc, FP=0xfffffe750c82bf00
CORE 5: PC=0xfffffe002e11e9e0, LR=0xfffffe002e11e9dc, FP=0xfffffe750cdcbf00
CORE 6: PC=0xfffffe002e11e9e0, LR=0xfffffe002e11e9dc, FP=0xfffffe750d1fbf00
CORE 7: PC=0xfffffe002e11e9dc, LR=0xfffffe002e11e9dc, FP=0xfffffe750c52bf00
Compressor Info: 101% of compressed pages limit (BAD) and 67% of segments limit (OK) with 38 swapfiles and OK swap space
Total cpu_usage: 102156909
Thread task pri cpu_usage
0xfffffe1ffec5f1c0 watchdogd 97 4067
0xfffffe1fffff31c0 kernel_task 91 3051756
0xfffffe1fffa1e8a0 kernel_task 91 3050345
0xfffffe1fff028100 thermalmonitord 37 15853
0xfffffe1ffef370c0 watchdogd 31 20852

Panicked task 0xfffffe1b33452108: 23 pages, 4 threads: pid 308: watchdogd
Panicked thread: 0xfffffe1ffec5f1c0, backtrace: 0xfffffe750c7fb170, tid: 3024
		  lr: 0xfffffe002e0e9100  fp: 0xfffffe750c7fb1e0
		  lr: 0xfffffe002e0e8dc8  fp: 0xfffffe750c7fb250
		  lr: 0xfffffe002e22d5c8  fp: 0xfffffe750c7fb270
		  lr: 0xfffffe002e21f878  fp: 0xfffffe750c7fb2e0
		  lr: 0xfffffe002e21d524  fp: 0xfffffe750c7fb3a0
		  lr: 0xfffffe002e0977f8  fp: 0xfffffe750c7fb3b0
		  lr: 0xfffffe002e0e8a50  fp: 0xfffffe750c7fb750
		  lr: 0xfffffe002e0e8a50  fp: 0xfffffe750c7fb7c0
		  lr: 0xfffffe002e90e5d4  fp: 0xfffffe750c7fb7e0
		  lr: 0xfffffe002eac3a94  fp: 0xfffffe750c7fb800
		  lr: 0xfffffe002eac3258  fp: 0xfffffe750c7fb820
		  lr: 0xfffffe002eac1d30  fp: 0xfffffe750c7fb940
		  lr: 0xfffffe002e86e390  fp: 0xfffffe750c7fbad0
		  lr: 0xfffffe002e1ed880  fp: 0xfffffe750c7fbbf0
		  lr: 0xfffffe002e0ef538  fp: 0xfffffe750c7fbc90
		  lr: 0xfffffe002e0c11fc  fp: 0xfffffe750c7fbcf0
		  lr: 0xfffffe002e0dc258  fp: 0xfffffe750c7fbd80
		  lr: 0xfffffe002e212994  fp: 0xfffffe750c7fbe50
		  lr: 0xfffffe002e21d8b4  fp: 0xfffffe750c7fbf10
		  lr: 0xfffffe002e0977f8  fp: 0xfffffe750c7fbf20
      Kernel Extensions in backtrace:
         com.apple.driver.AppleARMWatchdogTimer(1.0)[9E823012-9D79-3DE0-BF0B-8C290E00CA6D]@0xfffffe002eabf450->0xfffffe002eac3f33
            dependency: com.apple.driver.AppleARMPlatform(1.0.2)[12D5CDCA-8B0E-3783-8FEC-D4761DA26B1E]@0xfffffe002ea74db0->0xfffffe002eabf44b

last started kext at 908871355: com.apple.driver.driverkit.serial	6.0.0 (addr 0xfffffe002de80510, size 3416)
loaded kexts:
com.apple.filesystems.autofs	3.0
com.apple.driver.AppleTopCaseHIDEventDriver	5450.8
com.apple.driver.AppleBiometricServices	1
com.apple.driver.CoreKDL	1
com.apple.driver.AppleALSColorSensor	1.0.0d1
com.apple.driver.AppleAOPVoiceTrigger	140.1
com.apple.driver.DiskImages.ReadWriteDiskImage	493.0.0
com.apple.driver.DiskImages.UDIFDiskImage	493.0.0
com.apple.driver.DiskImages.RAMBackingStore	493.0.0
com.apple.driver.DiskImages.FileBackingStore	493.0.0
com.apple.driver.BCMWLANFirmware4378.Hashstore	1
com.apple.driver.SEPHibernation	1
com.apple.driver.AppleThunderboltIP	4.0.3
com.apple.driver.AppleUSBDeviceNCM	5.0.0
com.apple.driver.AppleSmartBatteryManager	161.0.0
com.apple.driver.AppleFileSystemDriver	3.0.1
com.apple.nke.l2tp	1.9
com.apple.filesystems.tmpfs	1
com.apple.AppleEmbeddedSimpleSPINORFlasher	1
com.apple.filesystems.lifs	1
com.apple.filesystems.apfs	1934.121.2
com.apple.IOTextEncryptionFamily	1.0.0
com.apple.filesystems.hfs.kext	583.100.10
com.apple.security.BootPolicy	1
com.apple.BootCache	40
com.apple.AppleFSCompression.AppleFSCompressionTypeZlib	1.0.0
com.apple.AppleFSCompression.AppleFSCompressionTypeDataless	1.0.0d1
com.apple.driver.AppleSmartIO2	1
com.apple.driver.ApplePMP	1
com.apple.driver.AppleCS42L83Audio	550.3
com.apple.driver.ApplePMPFirmware	1
com.apple.driver.AppleSPMIPMU	1.0.1
com.apple.driver.AppleTAS5770LAmp	550.3
com.apple.driver.AppleDPDisplayTCON	1
com.apple.driver.AppleAVE2	550.16.0
com.apple.driver.AppleMobileDispH13G-DCP	140.0
com.apple.driver.AppleT8020SOCTuner	1
com.apple.driver.AppleT8103CLPCv3	1
com.apple.driver.AppleAVD	564
com.apple.driver.AppleJPEGDriver	4.8.1
com.apple.AGXG13G	190.20
com.apple.driver.AudioDMAController-T8103	150.1
com.apple.driver.usb.AppleSynopsysUSB40XHCI	1
com.apple.driver.AppleSerialShim	1
com.apple.driver.AppleEventLogHandler	1
com.apple.driver.AppleS5L8960XNCO	1
com.apple.driver.AppleT8103PMGR	1
com.apple.driver.AppleS8000AES	1
com.apple.driver.AppleS8000DWI	1.0.0d1
com.apple.driver.AppleInterruptController	1.0.0d1
com.apple.driver.AppleT8020DART	1
com.apple.driver.AppleBluetoothModule	1
com.apple.driver.AppleSamsungSerial	1.0.0d1
com.apple.driver.AppleBCMWLANBusInterfacePCIe	1
com.apple.driver.AppleS5L8920XPWM	1.0.0d1
com.apple.driver.AppleS5L8940XI2C	1.0.0d2
com.apple.driver.AppleSPIMC	1
com.apple.driver.AppleT8101	1
com.apple.driver.AppleM68Buttons	1.0.0d1
com.apple.iokit.IOUserEthernet	1.0.1
com.apple.driver.usb.AppleUSBUserHCI	1
com.apple.iokit.IOKitRegistryCompatibility	1
com.apple.iokit.EndpointSecurity	1
com.apple.driver.AppleDiskImages2	126.100.13
com.apple.AppleSystemPolicy	2.0.0
com.apple.nke.applicationfirewall	402
com.apple.kec.InvalidateHmac	1
com.apple.kec.AppleEncryptedArchive	1
com.apple.driver.driverkit.serial	6.0.0
com.apple.kext.triggers	1.0
com.apple.iokit.IOAVBFamily	1040.6
com.apple.plugin.IOgPTPPlugin	1040.3
com.apple.iokit.IOEthernetAVBController	1.1.0
com.apple.driver.AppleHIDKeyboard	228.2
com.apple.driver.AppleHSBluetoothDriver	5450.8
com.apple.driver.IOBluetoothHIDDriver	9.0.0
com.apple.driver.AppleActuatorDriver	5450.14
com.apple.driver.AppleMultitouchDriver	5450.14
com.apple.driver.AppleMesaSEPDriver	100.99
com.apple.iokit.IOBiometricFamily	1
com.apple.driver.AppleAOPAudio	150.2
com.apple.driver.DiskImages.KernelBacked	493.0.0
com.apple.driver.AppleSEPHDCPManager	1.0.1
com.apple.driver.AppleTrustedAccessory	1
com.apple.iokit.AppleSEPGenericTransfer	1
com.apple.driver.AppleXsanScheme	3
com.apple.driver.usb.networking	5.0.0
com.apple.driver.AppleBTM	1.0.1
com.apple.driver.AppleThunderboltUSBDownAdapter	1.0.4
com.apple.driver.AppleThunderboltPCIDownAdapter	4.1.1
com.apple.driver.AppleThunderboltDPInAdapter	8.5.1
com.apple.driver.AppleThunderboltDPAdapterFamily	8.5.1
com.apple.driver.AppleConvergedIPCOLYBTControl	1
com.apple.driver.AppleConvergedPCI	1
com.apple.driver.AppleBluetoothDebug	1
com.apple.driver.AppleHIDTransportSPI	5450.4
com.apple.driver.AppleHIDTransport	5450.4
com.apple.driver.AppleInputDeviceSupport	5450.4
com.apple.driver.AppleDCPDPTXProxy	1.0.0
com.apple.nke.ppp	1.9
com.apple.driver.DCPDPFamilyProxy	1
com.apple.driver.AppleSPU	1
com.apple.driver.AppleBSDKextStarter	3
com.apple.driver.AppleDiagnosticDataAccessReadOnly	1.0.0
com.apple.filesystems.hfs.encodings.kext	1
com.apple.iokit.IONVMeFamily	2.1.0
com.apple.driver.AppleNANDConfigAccess	1.0.0
com.apple.AGXFirmwareKextG13GRTBuddy	190.20
com.apple.AGXFirmwareKextRTBuddy64	190.20
com.apple.driver.AppleCSEmbeddedAudio	550.3
com.apple.driver.AppleDialogPMU	1.0.1
com.apple.driver.AppleStockholmControl	1.0.0
com.apple.driver.AppleHPM	3.4.4
com.apple.driver.AppleEmbeddedAudio	550.3
com.apple.iokit.AppleARMIISAudio	140.2
com.apple.driver.DCPAVFamilyProxy	1
com.apple.driver.AppleSPMI	1.0.1
com.apple.iokit.IOMobileGraphicsFamily-DCP	343.0.0
com.apple.iokit.IOMobileGraphicsFamily	343.0.0
com.apple.driver.AppleDCP	1
com.apple.driver.AppleFirmwareKit	1
com.apple.driver.ApplePassthroughPPM	3.0
com.apple.driver.AppleSART	1
com.apple.driver.AppleH13CameraInterface	6.51.1
com.apple.driver.AppleH10PearlCameraInterface	17.0.6
com.apple.driver.AppleH11ANEInterface	5.65.2
com.apple.iokit.IOGPUFamily	35.29
com.apple.driver.AppleUSBXDCIARM	1.0
com.apple.driver.AppleUSBXDCI	1.0
com.apple.iokit.IOUSBDeviceFamily	2.0.0
com.apple.driver.usb.AppleSynopsysUSBXHCI	1
com.apple.driver.usb.AppleUSBXHCI	1.2
com.apple.driver.AppleEmbeddedUSBHost	1
com.apple.driver.usb.AppleUSBHub	1.2
com.apple.driver.usb.AppleUSBHostCompositeDevice	1.2
com.apple.driver.AppleT8103TypeCPhy	1
com.apple.driver.ApplePMGR	1
com.apple.driver.AppleARMWatchdogTimer	1
com.apple.driver.usb.AppleUSBHostPacketFilter	1.0
com.apple.driver.AppleDisplayCrossbar	1.0.0
com.apple.iokit.IODisplayPortFamily	1.0.0
com.apple.driver.AppleTypeCPhy	1
com.apple.driver.AppleThunderboltNHI	7.2.81
com.apple.driver.AppleT8103PCIeC	1
com.apple.iokit.IOThunderboltFamily	9.3.3
com.apple.driver.ApplePIODMA	1
com.apple.driver.AppleT8103PCIe	1
com.apple.driver.AppleMultiFunctionManager	1
com.apple.driver.AppleBluetoothDebugService	1
com.apple.driver.AppleBCMWLANCore	1.0.0
com.apple.iokit.IO80211Family	1200.12.2b2
com.apple.driver.IOImageLoader	1.0.0
com.apple.driver.AppleOLYHAL	1
com.apple.driver.corecapture	1.0.4
com.apple.driver.AppleEmbeddedPCIE	1
com.apple.driver.AppleMCA2-T8103	640.5
com.apple.driver.AppleEmbeddedAudioLibs	140.1
com.apple.driver.AppleFirmwareUpdateKext	1
com.apple.driver.AppleGPIOICController	1.0.2
com.apple.driver.AppleFireStormErrorHandler	1
com.apple.driver.AppleMobileApNonce	1
com.apple.iokit.IOTimeSyncFamily	1040.3
com.apple.driver.DiskImages	493.0.0
com.apple.iokit.IOGraphicsFamily	597
com.apple.iokit.IOBluetoothSerialManager	9.0.0
com.apple.iokit.IOBluetoothHostControllerUSBTransport	9.0.0
com.apple.iokit.IOBluetoothHostControllerUARTTransport	9.0.0
com.apple.iokit.IOBluetoothHostControllerTransport	9.0.0
com.apple.driver.IOBluetoothHostControllerPCIeTransport	9.0.0
com.apple.iokit.IOBluetoothFamily	9.0.0
com.apple.driver.FairPlayIOKit	68.16.0
com.apple.iokit.CSRBluetoothHostControllerUSBTransport	9.0.0
com.apple.iokit.BroadcomBluetoothHostControllerUSBTransport	9.0.0
com.apple.driver.AppleSSE	1.0
com.apple.driver.AppleSEPKeyStore	2
com.apple.driver.AppleUSBTDM	533.120.2
com.apple.iokit.IOUSBMassStorageDriver	210.120.3
com.apple.iokit.IOPCIFamily	2.9
com.apple.iokit.IOSCSIBlockCommandsDevice	456.121.3
com.apple.iokit.IOSCSIArchitectureModelFamily	456.121.3
com.apple.driver.AppleIPAppender	1.0
com.apple.driver.AppleFDEKeyStore	28.30
com.apple.driver.AppleEffaceableStorage	1.0
com.apple.driver.AppleCredentialManager	1.0
com.apple.driver.KernelRelayHost	1
com.apple.iokit.IOUSBHostFamily	1.2
com.apple.driver.AppleUSBHostMergeProperties	1.2
com.apple.driver.usb.AppleUSBCommon	1.0
com.apple.driver.AppleSMC	3.1.9
com.apple.driver.RTBuddy	1.0.0
com.apple.driver.AppleEmbeddedTempSensor	1.0.0
com.apple.driver.AppleARMPMU	1.0
com.apple.iokit.IOAccessoryManager	1.0.0
com.apple.driver.AppleOnboardSerial	1.0
com.apple.iokit.IOSkywalkFamily	1.0
com.apple.driver.mDNSOffloadUserClient	1.0.1b8
com.apple.iokit.IONetworkingFamily	3.4
com.apple.iokit.IOSerialFamily	11
com.apple.driver.AppleSEPManager	1.0.1
com.apple.driver.AppleA7IOP	1.0.2
com.apple.driver.IOSlaveProcessor	1
com.apple.driver.AppleBiometricSensor	2
com.apple.iokit.IOHIDFamily	2.0.0
com.apple.iokit.CoreAnalyticsFamily	1
com.apple.AUC	1.0
com.apple.iokit.IOAVFamily	1.0.0
com.apple.iokit.IOHDCPFamily	1.0.0
com.apple.iokit.IOCECFamily	1
com.apple.iokit.IOAudio2Family	1.0
com.apple.driver.AppleIISController	140.1
com.apple.driver.AppleAudioClockLibs	140.1
com.apple.driver.AppleM2ScalerCSCDriver	265.0.0
com.apple.iokit.IOSurface	302.14
com.apple.driver.IODARTFamily	1
com.apple.security.quarantine	4
com.apple.security.sandbox	300.0
com.apple.kext.AppleMatch	1.0.0d1
com.apple.driver.AppleMobileFileIntegrity	1.0.5
com.apple.security.AppleImage4	4.2.0
com.apple.kext.CoreTrust	1
com.apple.iokit.IOCryptoAcceleratorFamily	1.0.1
com.apple.driver.AppleARMPlatform	1.0.2
com.apple.iokit.IOStorageFamily	2.1
com.apple.iokit.IOSlowAdaptiveClockingFamily	1.0.0
com.apple.iokit.IOReportFamily	47
com.apple.kec.pthread	1
com.apple.kec.Libm	1
com.apple.kec.corecrypto	12.0



** Stackshot Succeeded ** Bytes Traced 1320529 (Uncompressed 3900192) **
```

**Comment by nexxeln** [comment]
That is *interesting*. Can anyone with an M1 mac reproduce and confirm this?

**Comment by Dhravya** [comment]
This is very interesting. Well, this might be because of some package we use?
I'll try to look into it

**Comment by Dhravya** [comment]
@matt123miller works fine with NPM?

**Comment by aslakhol** [comment]
I just did a quick test, and seems fine on my machine. 

MacBook Pro (16-inch, 2021)
Apple M1 Max
32 GB
macOS 12.4 (21F79)

yarn --version
1.22.19

**Comment by c-ehrlich** [comment]
Works for me. M1 MacBook Air, MacOS 12.2.1.

**Comment by juliusmarminge** [comment]
I am getting an error. Yarn version 1.22.15.

See here: https://app.warp.dev/block/3HbSoaLA3TdvBtOb5NVupu

**Comment by c-ehrlich** [comment]
> I am getting an error. Yarn version 1.22.15. M1 Pro with MacOS 12.4
> 
> See here: https://app.warp.dev/block/3HbSoaLA3TdvBtOb5NVupu

That looks like a completely different error to the one from the OP, and I can confirm I also get this when I'm running Node 18.

**Comment by matt123miller** [comment]
Ah, something I didn't mention. I got the same error as mentioned in the other issue. I would then run the command again which is when I got the crash.

**Comment by TheoBr** [comment]
Can you confirm that `yarn` is running as an arm process and not on Rosetta? My guess is some weird chaos between yarn in rosetta, old node versions, and prisma's native binaries failing

**Comment by matt123miller** [comment]
Ok my node is arm64
```
❯ node -v && yarn -v && node -p process.arch
v18.4.0
3.2.1
arm64
```

yarn is just running a big js file for the cli so presumably that's running under arm64 as well.

I'll try again using node 16.15.1 which I also have on my system, also arm64

```
❯ node -v && yarn -v && node -p process.arch
v16.15.1
3.2.1
arm64
```

I'll post a new comment shortly with the outcome because if it fails my mac becomes unusable and forces itself to shutdown.

**Comment by matt123miller** [comment]
Using npm with following details installed everything correctly, with the site booting up and working as intended.

```
node -v && npm -v && node -p process.arch
v18.4.0
8.12.1
arm64
```

I'm happy to use npm, I've only recently started playing with yarn instead because of it's new pnp zero installs approach. Perhaps I've uncovered an incompatibility using yarn, pnp mode, node 18 etc. or maybe I'm just unlucky. Either way I can continue. What happens with the issue and it's investigation is up to maintainers.

**Comment by sidwebworks** [comment]
Looks like an issue with `strict-peer` deps inside yarn 3 or something

**Comment by ariel-frischer** [comment]
Had the same issue running with yarn berry v3 `yarn create t3-app` but on my Manjaro arch system. `npx` method works fine though, so I'm sticking with it.

**Comment by nexxeln** [comment]
This is an issue with yarn and we can't really do anything about it. So closing this now.

**Comment by TheoBr** [comment]
Wait the unpin was accidental I thought that was a UI thing on my end

**Comment by c-ehrlich** [comment]
Pin might not be necessary now that https://github.com/t3-oss/create-t3-app/pull/173 is merged?

**Comment by nexxeln** [comment]
Yeah, I'll unpin thanks.

**Comment by MAST1999** [comment]
Hey, so I found a workaround that lets you use Yarn 3 and fixes this problem so I thought I'd post it here if others need it.

[Yarn PnP fails to resolve #ansi-styles](https://github.com/chalk/chalk/issues/531#issuecomment-1146705502)

I also don't know if it can cause other problems, so make of this what you will.

**Comment by juliusmarminge** [comment]
> Hey, so I found a workaround that lets you use Yarn 3 and fixes this problem so I thought I'd post it here if others need it.

Implemented in #173
